### ISSUE TITLE
prefer-lodash-compose

### ISSUE THREAD

**Issue by graingert** [wix/eslint-plugin-lodash#176]
```js
// bad
ham
  |> spam
  |> eggs


// good
_.flow(
    ham,
    spam,
    eggs,
)
```

basically the opposite of https://github.com/mysticatea/eslint-plugin/issues/19

**Comment by ganimomer** [comment]
Thanks for suggesting the rule!
I personally really look forward to the pipeline operator being a part of the spec, but as long as it's not supported by the `espree` parser, there's no way to detect it via ESLint rules.
Once it's supported, I think this would be a great idea (though maybe I'd integrate it into the `consistent-compose` rule), but for now, unfortunately, I'll have to close this.
